frontend/src/v5/ui/components/viewer/cards/card.styles.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { ScrollArea } from '@controls/scrollArea';
+import styled from 'styled-components';
+
+export const CardContainer = styled.div`
+	display: flex;
+
+
+	height: 100%;
+	flex-direction: column;
+
+	background: ${({ theme }) => theme.palette.primary.contrast};
+	border-radius: 10px;
+
+	margin-bottom: 20px;
+
+	box-shadow: ${({ theme }) => theme.palette.shadows.level_2};
+`;
+
+export const CardHeader = styled.div`
+	${({ theme }) => theme.typography.h3}
+	display: flex;
+	align-items: center;
+	height: 48px;
+	padding: 0px 15px;
+    border-bottom: 1px solid  ${({ theme }) => theme.palette.base.lightest};
+	gap: 4px;
+`;
+
+export const CardContent = styled(ScrollArea)`
+	& > div {
```

Comment:
This does not work as intended. Targeting all the direct children of the scrollbar which are `div`s results in padding added next to the scollbar. 
![image](https://user-images.githubusercontent.com/43547703/188138346-9a4cd971-e37f-4cc6-8f55-8966c972f2f4.png)

You should replace it with `& > :first-child` to fix it.

Also note that the scrollbar does not fade away when inactive

Reply:
ahh, ok. But I think I'll create a component for that. Targeting just the first child could create problems if the content consists of more than one children